generators/workspaces/generator.ts

```diff
@@ -21,10 +21,14 @@ import { existsSync } from 'fs';
 
 import { GENERATOR_ANGULAR, GENERATOR_BOOTSTRAP_WORKSPACES, GENERATOR_GIT, GENERATOR_REACT } from '../generator-list.js';
 
+import type { Config, Options } from '../base-workspaces/index.js';
 import BaseWorkspacesGenerator from '../base-workspaces/index.js';
 import { packageJson } from '../../lib/index.js';
 
-export default class WorkspacesGenerator extends BaseWorkspacesGenerator<{ baseName: string; monorepository: boolean }> {
+export default class WorkspacesGenerator extends BaseWorkspacesGenerator<
+  Config & { baseName: string; monorepository: boolean },
```

Comment:
Is it possible to put the full type it in api.d to follow convention of base-application?

Reply:
should be in types.d.ts.